chromium_src/chrome/browser/ui/commander/bookmark_command_source.cc

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/chromium_src/chrome/browser/ui/commander/bookmark_command_source.h"
+
+#include "src/chrome/browser/ui/commander/bookmark_command_source.cc"  // IWYU pragma: export
+
+#include "components/grit/brave_components_strings.h"
+#include "ui/base/l10n/l10n_util.h"
+
+namespace commander {
+
+BraveBookmarkCommandSource::BraveBookmarkCommandSource() = default;
+BraveBookmarkCommandSource::~BraveBookmarkCommandSource() = default;
+
+CommandSource::CommandResults BraveBookmarkCommandSource::GetCommands(
+    const std::u16string& input,
+    Browser* browser) const {
+  CommandSource::CommandResults results;
+  bookmarks::BookmarkModel* model =
+      BookmarkModelFactory::GetForBrowserContext(browser->profile());
+  // Just no-op instead of waiting for the model to load, since this isn't
+  // a persistent UI surface and they can just try again.
+  if (!model || !model->loaded() || !model->HasBookmarks()) {
+    return results;
+  }
+
+  FuzzyFinder finder(input);
+  std::vector<gfx::Range> ranges;
+  std::u16string open_title =
+      l10n_util::GetStringUTF16(IDS_COMMANDER_OPEN_BOOKMARK);
```

Comment:
I assume that all code of this method is copied to replace this line because it's difficult to replace it w/o patching.
Curious we prefer copying whole imples over one-line patching? @brave/chromium-src-reviewers

Reply:
Additionally, I think upstream is considering removing this code & the related feature

Reply:
ah, got it.